src/pages/Projects/ProjectDetail/ProjectDetail.tsx

```diff
@@ -68,7 +68,7 @@ export function ProjectDetail({
 
   return (
     <div>
-      <ProjectsHeader name={project?.name} />
+      <ProjectsHeader name={project?.name} project={project} />
```

Comment:
It feels a bit redundant to have both the `project` and `name` as parameters. Can we change the `name` parameter to `title` so it is more general? And we can still pass in the project name like:
`<ProjectsHeader title={project?.name} project={project} />`

Reply:
I removed the name prop entirely, I realized it was unnecessary.